components/forms/w-field.vue

```diff
@@ -1,9 +1,9 @@
 <template>
-  <component :is="as" class="field" :class="{ 'is-invalid': hasErrorMessage, 'is-disabled': disabled, [$attrs.class || '']: true }" :role="role" v-bind="wrapperAria">
-    <component :is="labelType" v-if="label" class="field-label" :id="labelId" :for="labelFor" :role="valueOrUndefined(labelLevel, 'heading')" :aria-level="valueOrUndefined(labelLevel, labelLevel)">{{ label }}<span v-if="optional" class="pl-8 font-normal text-14 text-gray-500"> (valgfritt)</span></component>
+  <component :is="as" class="relative" :class="{[input.invalid]: hasErrorMessage, [$attrs.class || '']: true}" :role="role" v-bind="wrapperAria">
```

Comment:
> `class="relative"`

Shouldn't that be moved to component-classes, too? I'm merely looking at the original [field](https://github.com/fabric-ds/css/blob/next/src/components/field.css#L1-L3) style.

Reply:
Yes, let's do it